src/app/contexts/EventContext/useWindowEvent/index.js

```diff
@@ -0,0 +1,9 @@
+export const useWindowEvent = (event, handler, passive = false) => () => {
```

Comment:
Seems like there is 2 exports of the same function in this file - a named export and a default export of `useWindowEvent`

Reply:
Fixed!